### DISCUSSION TITLE
Use displays in the Calendar layout

### DISCUSSION THREAD

**Discussion by thepurpleblob** [directus/directus#11894] [Feature Requests]
Calendar view shows colors as the hex code, not a colored dot (assuming dot option is selected)

### To Reproduce

Create a new table add a date field and a color field
Configure the color field to display as a coloured dot
Add some data
Show the table contents in list layout - the color is shown as a coloured dot. 
Change to calendar view. The colour is shown as the hex value, not the actual color. 

### What version of Directus are you using?

9.5.1

### What version of Node.js are you using?

16.13.2

### What database are you using?

SQLite

### What browser are you using?

Chrome

### What operating system are you using?

Linux 5.4.0-97-generic

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Technically this is not a bug, since currently the calendar layout passes the value as string because [FullCalendar's Event Object](https://fullcalendar.io/docs/event-object), particular `title` property, is just a string and can't (at least natively) show any displays like in Table layout. Kindly refer https://github.com/directus/directus/issues/9560#issuecomment-962507823 for more info on a related issue.

Maybe a potential enhancement would have something to do with [`eventBackgroundColor`](https://fullcalendar.io/docs/eventBackgroundColor).

**Reply by amo42** [reply]
That would be super handy. I could image to have the BG color set by a color-field from the related collection item. So this would be super handy in case you have multiple categories to visualize within on calendar. Technically, this is a different approach, but I assume it is related to the feature of rending the status color to the title.

**Comment by thepurpleblob** [comment]
Understood - but from an end-user POV it does seem inconsistent (with the list layout)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
